src/app/[locale]/(pages)/accedi/page.tsx

```diff
@@ -38,6 +40,13 @@ const Access = (): React.ReactElement => {
     value: process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L3,
   };
 
+  useEffect(() => {
+    if (isBrowser()) {
+      initAnalytics();
+      trackEvent('IO_LOGIN');
+    }
+  }, [isBrowser()]);
```

Comment:
Does initialize Mixpanel in the "accedi" page cover all the possible edge cases? I mean, if we, during a flow, don't navigate here but in another page, there's the possibility not to track some events. Am I right?

Should we do better initialize Mixpanel in the header component, or in another common component, paying attention not to initialize it twice? What do you think about that?

Reply:
Yes, I placed it there just for testing, it wouldn't have been the final position. For now, I'm putting it in the header using useMemo to render it only once. Let see if it works